frontend/src/features/performance/PerformanceInitial.tsx

```diff
@@ -0,0 +1,137 @@
+
+import { useInitialStore } from "./stores/useInitial";
+import { useState, useEffect } from "react";
+import PerformanceInitialEnduranceChart from "./PerformanceInitialChart";
+import { usePower } from "../power_unit/hooks/usePower";
+import InputNumber from "../common/inputs/InputNumber";
+import InputSlider from "../common/inputs/InputSlider";
+
+const PerformanceInitial = () => {
+  const method_type = useInitialStore((state) => state.method_type)
+  const proptype = useInitialStore((state) => state.proptype)
+  const propnumber = useInitialStore((state) => state.propnumber)
+  const nominalPower = useInitialStore((state) => state.nominalPower)
+  const flightAltitude = useInitialStore((state) => state.flightAltitude)
+  const startMass = useInitialStore((state) => state.startMass)
+  const fuelMass = useInitialStore((state) => state.fuelMass)
+  const fuelcons = useInitialStore((state) => state.fuelcons)
+  const vmax = useInitialStore((state) => state.vmax)
+  const wmax = useInitialStore((state) => state.wmax)
+  const area  = useInitialStore((state) => state.area)
+  const aspectRatio = useInitialStore((state) => state.aspectRatio)
+  const cx0 = useInitialStore((state) => state.cx0)
+  const czmax = useInitialStore((state) => state.czmax)
+  const setProptype = useInitialStore((state) => state.setProptype)
+  const setPropnumber = useInitialStore((state) => state.setPropnumber)
+  const setNominalPower = useInitialStore((state) => state.setNominalPower)
+  const setFlightAltitude = useInitialStore((state) => state.setFlightAltitude)
+  const setStartMass = useInitialStore((state) => state.setStartMass)
+  const setFuelMass = useInitialStore((state) => state.setFuelMass)
+  const setFuelcons = useInitialStore((state) => state.setFuelcons)
+  const setVmax = useInitialStore((state) => state.setVmax)
+  const setWmax = useInitialStore((state) => state.setWmax)
+  const setArea = useInitialStore((state) => state.setArea)
+  const setAspectRatio = useInitialStore((state) => state.setAspectRatio)
+  const setCx0 = useInitialStore((state) => state.setCx0)
+  const setCzmax = useInitialStore((state) => state.setCzmax)
+
+  const [calculatePower] = usePower();
+  const [maxPower, setMaxPower] = useState(calculatePower(0))
+
+  useEffect(() => {
+    setMaxPower(() => calculatePower(flightAltitude))
+  }, [flightAltitude, calculatePower])
+
+  return (
+    <div className="flex w-full p-4">
+      <div className="flex flex-col w-80 mr-8 space-y-1">
+        <InputSlider
+          value={propnumber}
+          setter={setPropnumber}
+          step={1}
+          min={1}
+          max={2}
+          label="Number of Engines"
+          unit=""
+        />
+        <InputNumber
```

Comment:
We need to integrate it with Power Unit Results. There is one catch though. We will need to interpolate the power for a given speed for fixed propellers, as there is no straight forward way to calculate it directly

Reply:
Agreed to postpone integration of power calculations between the modules.